source/features/forked-to.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import cache from '../libs/cache';
+import {getRepoURL} from '../libs/utils';
+import {isOwnRepo} from '../libs/page-detect';
+
+const getCacheKey = (repo: string): string => `forked-to:${repo}`;
+
+async function showForks(): Promise<void> {
+	const cached = await getValidatedCache(getSourceRepo());
+	const pageHeader = select('.pagehead h1.public')!;
+	for (const fork of cached.filter(fork => fork !== getRepoURL())) {
+		pageHeader.append(
+			<span className="fork-flag rgh-forked">forked to&nbsp;
+				<a href={`/${fork}`}>{fork}</a>
+			</span>
+		);
+	}
+}
+
+function rememberCurrentFork(): void {
+	if (!isOwnRepo()) {
+		return;
+	}
+
+	const forkedRepo = findForkedRepo();
+	if (forkedRepo) {
+		addAndStoreCache(forkedRepo, getRepoURL());
+	}
+}
+
+function watchForkDialog(): void {
+	const forkDialog = select('details-dialog[src*="/fork"]')!;
+	select('include-fragment', forkDialog)!.addEventListener('load', () => {
+		const forks = select.all('.octicon-repo-forked', forkDialog).map(forkElement => {
+			return forkElement.parentNode!.textContent!.trim();
+		});
+		addAndStoreCache(getSourceRepo(), ...forks);
+	});
+}
+
+function findForkedRepo(): string | undefined {
+	const forkSourceElement = select<HTMLAnchorElement>('.fork-flag:not(.rgh-forked) a');
+	if (forkSourceElement) {
+		return forkSourceElement.pathname.substring(1);
```

Comment:
```suggestion
		return forkSourceElement.pathname.slice(1);
```

Just my preference. https://github.com/sindresorhus/eslint-plugin-unicorn/issues/65